project/MimaExcludes.scala

```diff
@@ -122,6 +122,13 @@ object MimaExcludes {
     // [SPARK-41072][SS] Add the error class STREAM_FAILED to StreamingQueryException
     ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.sql.streaming.StreamingQueryException.this"),
 
+
+    // [SPARK-41192] add taskIndex in SparkListenerSpeculativeTaskSubmitted Event
+    ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.scheduler.SparkListenerSpeculativeTaskSubmitted.apply"),
```

Comment:
Do we still need to add these mima filters?

Reply:
Fixed ~